### ISSUE TITLE
Wireshark v4.2.4-1 does not launch with Qt v6.7.0-1.

### ISSUE THREAD

**Issue by KSR-Yasuda** [msys2/MINGW-packages#20658]
### Description / Steps to reproduce the issue

Execute `C:\msys64\mingw64\bin\wireshark.exe` from Explorer,
it shows an error dialog as below:

```text
---------------------------
wireshark.exe - エントリ ポイントが見つかりません
---------------------------
プロシージャ エントリ ポイント _ZN10QAudioSink12stateChangedEN6QAudio5StateE が
ダイナミック リンク ライブラリ C:\msys64\mingw64\bin\wireshark.exe から見つかりませんでした。
---------------------------

# Translation
---------------------------
wireshark.exe - Entry point not found
---------------------------
Procedure entry point _ZN10QAudioSink12stateChangedEN6QAudio5StateE
was not found in the dinamic link library C:\msys64\mingw64\bin\wireshark.exe .
---------------------------
```

From bash console, it quits silently without the dialog.

```bash
% pacman -Q | grep -P 'wireshark|-qt6-'
mingw-w64-x86_64-qt6-5compat 6.7.0-1                    # Wireshark depending package
mingw-w64-x86_64-qt6-base 6.7.0-1                       # Wireshark depending package
mingw-w64-x86_64-qt6-multimedia 6.7.0-1                 # Wireshark depending package
mingw-w64-x86_64-qt6-multimedia-ffmpeg 6.7.0-1
mingw-w64-x86_64-qt6-shadertools 6.7.0-1
mingw-w64-x86_64-wireshark 4.2.4-1
```

It works with the `mingw-w64-x86_64-qt6-*` packages of the previous v6.6.3-1.

It would be because of the QT v6.7 bug below:

* [[QTBUG-123997] Public ABI breakage between 6.6 and 6.7 for QAudioSink::stateChanged(QtAudio::State)](https://bugreports.qt.io/browse/QTBUG-123997) , Qt Bug Tracker
* [wireshark-qt 4.2.4-1: "symbol lookup error: wireshark: undefined symbol: _ZN10QAudioSink12stateChangedEN6QAudio5StateE, version Qt_6" with testing repos enabled (#2)](https://gitlab.archlinux.org/archlinux/packaging/packages/wireshark/-/issues/2), Issue, Arch Linux / Packaging / Packages / wireshark, GitLab
    - It says `wireshark-qt 4.2.4-2` solved the issue.

### Expected behavior

Wireshark successfully launches with the latest Qt packages.

### Actual behavior

Wireshark v4.2.4-1 launch fails with Qt v6.7.0-1 packages.

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

MINGW64_NT-10.0-22631

### MINGW environments affected

- [X] MINGW64
- [ ] MINGW32
- [ ] UCRT64
- [ ] CLANG64
- [ ] CLANG32
- [ ] CLANGARM64

**Comment by Biswa96** [comment]
Thank you for reporting the issue. I can reproduce it in my system.

![image](https://github.com/msys2/MINGW-packages/assets/31443074/29d7a7c0-d4f0-45c9-b22f-4448c6ab4317)

**Comment by Biswa96** [comment]
Fixed in 632e335af9eb7b3c0471581767aad3944582c9e1

**Comment by KSR-Yasuda** [comment]
Thank you!
Now the latest Wireshark v4.2.4-2 works with the latest Qt v6.7.0-1.
